lib/MusicBrainz/Server/Data/Collection.pm

```diff
@@ -136,33 +136,51 @@ sub get_first_collection {
     return $self->sql->select_single_value($query, $editor_id);
 }
 
-sub find_all_by_editor {
```

Comment:
Huh. After moving the `ORDER BY` back to the inner query, I can't reproduce whatever ordering issues I was running into. I think assumed that was the case since I couldn't find anywhere in the docs that the ordering was guaranteed to be the same. So either I missed something or postgres's implementation does guarantee that. Not a satisfying conclusion. :)